src/components/ChatWindow.tsx

```diff
@@ -44,6 +46,9 @@ class ChatWindow extends React.Component<Props, State> {
   };
 
   componentDidMount() {
+    const {API_BASE_URL, WS_URL} = getBackendUrl()
+    this.setState({API_BASE_URL});
```

Comment:
if we put this `baseUrl` on props, we could use the results of `getBackendUrl` as the fallback value if the user doesn't specify a custom url